### DISCUSSION TITLE
New Update - "Out of date!" warning is a bit Heavy

### DISCUSSION THREAD

**Discussion by SimplSam** [d2phap/ImageGlass#932] [General] [answered]
Hi. Can we have a more subtle - "New Update Available" message - rather than the **"ImageGlass is out of date!**" warning message?

The current message looks like something we would expect if there was problem with the app ( which there never is :) ).

Also - the current references and link to the Changelog on the Download page are incorrect. They reference: 8.0.8.12, but should be 8.0.12.8

**Comment by d2phap** [comment, marked as answer]
Thanks @SimplSam 
I will surely update this in the next versions!

**Comment by SimplSam** [comment]
Thanks. As mentioned the references to the new release on the website are incorrectly numbered: i.e. "ImageGlass 8.0.8.12 - Kobe", which should be "ImageGlass 8.0.**12.8** - Kobe" etc. etc.

https://imageglass.org/
https://imageglass.org/download

**Reply by d2phap** [reply]
Thanks! I fixed the version!
